app/views/teaching_events/index.html.erb

```diff
@@ -33,42 +33,42 @@
       <%= render partial: "teaching_events/index/filter" %>
     </div>
 
-    <% if @featured_events.blank? && @events.blank? %>
-      <%= render partial: "teaching_events/index/no_results" %>
-    <% else %>
-      <div class="teaching-events__listing">
-        <% if @featured_events.any? %>
-          <div class="teaching-events__events teaching-events__events--featured">
-            <h2 class="heading-m">Upcoming Get Into Teaching events</h2>
+    <div class="teaching-events__listing">
+      <% if @featured_events.blank? && @events.blank? %>
+        <%= render partial: "teaching_events/index/no_results" %>
+      <% end %>
 
-            <ol><%= render TeachingEvents::EventComponent.with_collection(@featured_events) %></ol>
+      <% if @featured_events.any? %>
```

Comment:
It could be but I think it makes it more confusing because you end up with this (as 2/3 blocks will apply if there are events):

```
if @featured_events.blank && @events.blank?
elsif @featured_events
end

if @events
end
```

_Technically_ we should perhaps do:

```
if @featured_events.blank? && @events.blank?
else
  if @featured_events
   end

  if @events
   end
end
```

Both of which I think are less clear than what we have though:

```
if @featured_events.blank? && @events.blank?
end

if @featured_events
end

if @events
end
```

It's sometimes a shame that Ruby's case statements don't fall through 😅

Reply:
👍 - That's find. Just keep it as it is.